### PR TITLE
Adding example gallery + minor refactor of the doc layout 

### LINKED ISSUE #103: Add an example gallery
### Describe the feature

add one, it would be good to show how to query for different data etc. 
And in the future if more things are added

### PR DESCRIPTION
**Pull request by hayesla**
This draft PR is to add an example gallery to the docs

Fixes #103 

@nabobalis I may need your help with the sphinx stuff 🤷‍♀️

### CODE REVIEW

**Comment by dstansby** [comment]
Looks like you're missing sphinx-gallery from the extensions in `conf.py` too: https://sphinx-gallery.github.io/stable/getting_started.html#configure-and-use-sphinx-gallery

**Comment by hayesla** [comment]
thanks @dstansby  - looks like thats working now

File: setup.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = A sunpy FIDO plugin for accessing data in the Solar Orbiter Archiv
 long_description = file: README.rst
 author = David Stansby and The SunPy Community
 author_email = sunpy@googlegroups.com
-url =
+url = https://sunpy.org
```
Comment (nabobalis) [inline setup.cfg:7]:
```suggestion
url = https://github.com/sunpy/sunpy-soar
```
or maybe the docs is better?

File: examples/example_template/example_template.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+"""
```
Comment (nabobalis) [inline examples/example_template/example_template.py:1]: Trying to work out why this example is missing from the rendered docs.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**